fvm/tests/dummy.rs

```diff
@@ -201,6 +203,10 @@ impl Machine for DummyMachine {
     fn new_limiter(&self) -> Self::Limiter {
         DummyLimiter::default()
     }
+
+    fn commit_events(&self, _events: &[StampedEvent]) -> kernel::Result<Option<Cid>> {
+        todo!()
```

Comment:
Implementing this as a helper function would let you skip these todo's. (or you could give them a default implementation, up to you).

Reply:
Ack but https://github.com/filecoin-project/ref-fvm/pull/1049#discussion_r1023165039.